src/common/file.c

```diff
@@ -58,6 +58,8 @@
 
 #define DEVICE_DAX_ZERO_LEN (2 * MEGABYTE)
 
+int Fallc = 0;
```

Comment:
`Fallocate_at_create`
This is a global variable, and should be named descriptively.